src/input/Input.tsx

```diff
@@ -66,13 +90,8 @@ export const InputField = forwardRef(
       >
         {prefix && <span>{prefix}</span>}
         {type === "textarea" ? (
-          <textarea
-            id={id}
-            {...rest}
-            ref={ref}
-            type={type}
-            disabled={disabled}
-          />
+          // for review: can i remove type and ref here?
```

Comment:
I don't think so, the types will hopefully work out if you change the type of `ref` to `HTMLInputElement | HTMLTextAreaElement`

Reply:
That is what I tried first. It throws the following error though:
```
Type 'ForwardedRef<HTMLInputElement | HTMLTextAreaElement>' is not assignable to type 'LegacyRef<HTMLTextAreaElement>'.
  Type 'MutableRefObject<HTMLInputElement | HTMLTextAreaElement>' is not assignable to type 'LegacyRef<HTMLTextAreaElement>'.
    Type 'MutableRefObject<HTMLInputElement | HTMLTextAreaElement>' is not assignable to type 'RefObject<HTMLTextAreaElement>'.
      Types of property 'current' are incompatible.
        Type 'HTMLInputElement | HTMLTextAreaElement' is not assignable to type 'HTMLTextAreaElement'.
          Type 'HTMLInputElement' is missing the following properties from type 'HTMLTextAreaElement': cols, rows, textLength, wrapts(2322)
index.d.ts(143, 9): The expected type comes from property 'ref' which is declared here on type 'DetailedHTMLProps<TextareaHTMLAttributes<HTMLTextAreaElement>, HTMLTextAreaElement>'
```

Reply:
I am doing an explicit cast for the ref now. I think that should work.